CONTRIBUTING.md

```diff
@@ -18,9 +18,9 @@ We have sub projects in each branch:
 |      branch    | purpose |
 |----------------|---------|
 |   master       | Singleton service code  |
-|   java-client  | Singleton java client code  |
-|   js-client    | Singleton js client code  |
-|   devops       | CI and automation testing code, it is not place to contribute code unless you want to add CI or test scripts.  |
+|   java-client  | Singleton Java client code |
+|   js-client    | Singleton JavaScript client code |
```

Comment:
It seems the branch name here does not exactly match the branch name in the repo and we also need to add the "g11n-angular-client" branch here, could you take a look?